src/dropdownMenu/dropdownMenu.js

```diff
@@ -4,70 +4,79 @@ angular.module('mm.foundation.dropdownMenu', [])
     return {
         bindToController: {
             disableHover: '=',
-            disableClickOpen: '='
+            disableClickOpen: '=',
+            closingTime: '='
         },
         scope: {},
         restrict: 'A',
         controllerAs: 'vm',
         controller: function($scope, $element) {
             'ngInject';
             var vm = this;
-            // $element is-dropdown-submenu-parent
         }
     };
 })
-.directive('li', () => {
-    return {
+.directive('li', ['$timeout', dropdownMenuItem]);
```

Comment:
We're using angular-annotate so there is no need for setting $inject manually. Could you instead add 'ngInject'; on the first line of the body of dropdownMenuItem?